mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -689,4 +689,29 @@ public void runVoid() throws MailboxException {
         );
     }
 
+    @Override
```

Comment:
We need some tests about this, I believe. You can write tests with mockito, I believe.